packages/theme-dark-extension/style/variables.css

```
@@ -436,4 +436,8 @@ all of MD as it is not optimized for dense, information rich UIs.
   --jp-switch-color: var(--md-grey-400);
   --jp-switch-true-position-color: var(--md-orange-700);
   --jp-switch-cursor-color: rgba(0, 0, 0, 0.8);
+
+  /* Variables for the menu font color : to highlight the active items */
+  --jp-menu-active-font-color: var(--jp-ui-font-color0);
+  --jp-menu-inactive-font-color: var(--jp-ui-font-color3);
```

The naming does not seem accurate to me. The active item is `.lm-Menu-item.lm-mod-active`.

```suggestion
  /* Variables for the menu font color */
  --jp-menu-enabled-font-color: var(--jp-ui-font-color0);
  --jp-menu-disabled-font-color: var(--jp-ui-font-color3);
```